src/utils/__test__/formHelperFunctions.spec.js

```diff
@@ -37,7 +37,8 @@ describe('formHelperFunctions', () => {
               name: 'Name 1',
               metadata: {}
             },
-            hide_ifs: []
+            hide_ifs: [],
+            show: false
```

Comment:
i left a coment on jay's PR as well -- IMO show is a little generic, could we maybe call it like show_logic?